postgis_source.go

```diff
@@ -53,21 +54,20 @@ func (c *PostGISConfig) Dataset() (*goqu.SelectDataset, error) {
 		return nil, InvalidTableConfig
```

Comment:
Nit: Might be nice to also change the comment in L49 (can't annotate there) to no longer talk about "CTE-based SelectDataset"

Reply:
Good call (I believe this is what @madhulikajc was referring to as well). I'll update the comment!